src/test/java/com/uid2/admin/vertx/EncryptionKeyServiceKeysetsDisabledTest.java

```diff
@@ -148,32 +145,6 @@ void addSiteKey() throws Exception {
     }
 
```

Comment:
why are these deleted?

Reply:
The method they were testing has stopped being used a while back.